internal/indexer/postgres/packagesbylayer.go

```diff
@@ -102,7 +96,7 @@ func packagesByLayer(ctx context.Context, db *sqlx.DB, hash claircore.Digest, sc
 			&pkg.Module,
 			&pkg.Arch,
 
-			&spkg.ID,
+			&srcID,
```

Comment:
was this a bug previously?

Reply:
I think so. I think we only didn't notice because of our... weak support for source packages.

Reply:
yeah think so lol.